tests/test_procs.py

```diff
@@ -0,0 +1,20 @@
+"""
```

Comment:
This file should probably be `tests/procs/test_pipelines.py`

Reply:
Done